aacgmv2/__init__.py

```diff
@@ -47,10 +48,14 @@
 high_alt_trace = 6378.0  # 1 RE, these are ionospheric coordinates
 
 # Path and filename prefix for the IGRF coefficients
-AACGM_v2_DAT_PREFIX = _os.path.join(_os.path.realpath(
-    _os.path.dirname(__file__)), 'aacgm_coeffs', 'aacgm_coeffs-13-')
-IGRF_COEFFS = _os.path.join(_os.path.realpath(_os.path.dirname(__file__)),
-                            'magmodel_1590-2020.txt')
+AACGM_v2_DAT_PREFIX = _os.path.join(
+    _os.path.split(str(resources.path(__package__,
+                                      '__init__.py').__enter__()))[0],
+    'aacgm_coeffs', 'aacgm_coeffs-14-')
+IGRF_COEFFS = _os.path.join(
+    _os.path.split(str(resources.path(__package__,
+                                      '__init__.py').__enter__()))[0],
+    'magmodel_1590-2025.txt')

```

Comment:
These are the lines that generate the Deprecation Warning. `resources.path` is replaced by `resources.files`. Probably worth an issue for the next update.

Reply:
#94